Hackney.Shared.Finance/Boundary/Response/ApiResponse.cs

```diff
@@ -0,0 +1,25 @@
+﻿
+namespace Hackney.Shared.Finance.Boundary.Response
+{
+    /// <summary>
+    /// API Response wrapper for all API responses
+    /// If a request has been successful this will be denoted by the statusCode
+    ///     Then the 'data' property will be populated
+    /// If a request has not been successful denoted
+    ///     Then the Error property will be populated
+    /// </summary>
+    /// <typeparam name="T"></typeparam>
+    public class ApiResponse<T> where T : class
+    {
+        public T Results { get; set; }
+
+        public long Total { get; set; }
+
+        public ApiResponse() { }
+
+        public ApiResponse(T result)
+        {
+            Results = result;
+        }
```

Comment:
Actually never mind the comment above, this seems like this may be some default code from some Nuget template. Or am I wrong?